_faqs/M1Workaround.md

```diff
@@ -2,34 +2,56 @@
 title: How to Install BciPy on Mac M1 Chip 
 ---
 
-If you are using a computer with the Mac M1 chip, you will run into issues with installing some packages of BciPy including numpy and psychopy. 
+If you are using a computer with the Mac M1 chip, you may run into issues with installing some packages of BciPy including psychopy and pyqt5. 
```

Comment:
We should note the versions this applies to. Let's specify versions `2.0.0rc1 - 2.0.0rc4`. Due to several dependencies, earlier versions likely will not work on m1 without significant manual effort (if at all), and we don't want to go back and test all of them. :)  We may fix some of these before releasing the rc4, but we can update the text later.